css/style.css

```
@@ -16,4 +16,129 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* This is Header (Navigation & Hero )*/
+
+/* Navigation */
+nav{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:140px;
+    width: 80%;
+    margin: auto;
+    font-size: 2.5rem;
+    }
+
+a{
+    height: 60px;
+    margin-left: 80px;
+    text-decoration: none;
+    color: rgb(65, 63, 63);
+    }
+
+a:hover{
+    color: rgb(250, 115, 4);
+    }
+
+.logo{
+    width:50px;
+    height:auto;
+    margin-right: auto;
+    margin-left: 0;
+    }
+
+.logo img{
+    width: 100%;
+    height: auto;
+    }
+
+ /* Hero style */
+.hero-image{
+    background-image: url(../img/first-background.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 900px;
+    position:relative;
+}
+
+.hero-text{
+    text-align:center;
+    position: absolute;
+    color: white;
+    top:20%;
+    left:40%;
+}
```

```suggestion
.hero-text{
    background-image: url(../img/first-background.jpg);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    width: 100%;
    height: 900px;
    
    text-align:center;
    color: white;
    
    /* use flexbox to centre the text and buttons */
    display: flex;
    flex-direction:column;
    align-items: center;
    justify-content: center;
}
```
Instead of using `position: absolute` to position it in the centre, a better way for aligning things in the centre is to use flexbox